app/views/application/tips.html.erb

```diff
@@ -0,0 +1,85 @@
+<main>
+  <%= render "header" %>
```

Comment:
Is it expected that theres some inconsistency to where the header is rendered? It seems like sometimes its inside the main, like here, sometimes theres no main, and sometimes the main encompasses the whole page? I'm surprised to see the differences, is there a reason?